### ISSUE TITLE
Integration with command-not-found

### ISSUE THREAD

**Issue by konaya** [nvbn/thefuck#417]
```
konaya@wulfenite:/tmp> time md5sum NAND.BIN 
time may be found in the following packages:
  extra/time 1.7-7                  /usr/bin/time
  community/plan9port 20150629-1    /usr/lib/plan9/bin/time
konaya@wulfenite:/tmp> fuck
No fucks given
```

Would it be possible to have thefuck work in situations like the one above?

(I'm running Arch.)

**Comment by mcarton** [comment]
On Arch it's already integrated but it requires `pkgfile` from the package `extra/pkgfile` to be installed. What do you use to get your *“time may be found in the following packages”* if not `pkgfile`?
(What kind of a shell does not have a built-in `time`? :open_mouth:)

**Comment by mcarton** [comment]
Oh, I see :sweat_smile: 
Are you using `fish`? If the problem is what I think, you're a very unfortunate guy:
  * `fish` does not have a `time` built-in;
  * `thefuck` executes every program with `sh` in order to get a consistent output no mater what the shell;
  * `sh` does have a `time` built-in;
  * then, when `thefuck` executes your command, `sh` executes it happily;
  * therefore there's no reason to propose you to install `time`, because `sh` did not tell us there is no such thing as `time`.

TL;DR: because we want a consistent output whatever the user's shell is, in that very specific case, we don't get a consistent output.

Can you confirm that you do have the package `extra/pkgfile` and get something like that:
```
~> eclipse
eclipse may be found in the following packages:
  extra/eclipse 4.4.2-1	/usr/bin/eclipse
~> fuck
yaourt -S extra/eclipse; and eclipse [enter/↑/↓/ctrl+c]
```
for any program you don't have that does not happen to be an `sh` built-in? :smile:

**Comment by konaya** [comment]
Ha! You're right, it actually works with other programs. Unlucky indeed! Not sure if it's worth fixing, really.

**Comment by scorphus** [comment]
Implementation works, as confirmed by user. And recent releases of Fish implement time as builtin.
